src/oss/paths/backup_shards_shardID.yml

```diff
@@ -7,10 +7,10 @@ get:
     - $ref: "../../common/parameters/TraceSpan.yml"
     - in: header
       name: Accept-Encoding
-      description: The Accept-Encoding request HTTP header advertises which content encoding, usually a compression algorithm, the client is able to understand.
+      description: The content-encoding directive, either `gzip` for compressed data or `identity` for unmodified, uncompressed data.
```

Comment:
`accept-encoding` and `content-encoding` are different things, are these typos?

Reply:
@glinton Not typos, but I've tried to clarify the headers and the schema descriptions further.